.buildkite/dagster-buildkite/dagster_buildkite/steps/helm.py

```diff
@@ -38,14 +39,14 @@ def build_helm_steps() -> List[BuildkiteStep]:
     ]
 
 
-def _build_lint_steps() -> List[CommandStep]:
+def _build_lint_steps(package_spec) -> List[CommandStep]:
     return [
         CommandStepBuilder(":yaml: :lint-roller:")
         .run(
             "pip install yamllint",
             "make yamllint",
         )
-        .with_skip(skip_if_no_helm_changes())
+        .with_skip(skip_if_no_helm_changes() or package_spec.skip_reason)
```

Comment:
I think we want to run this if there are no helm changes and we're skipping the package_spec, right?

Reply:
Oh yes you're absolutely right.